conda/plugins/manager.py

```diff
@@ -657,6 +664,84 @@ def get_environment_specifier(
         else:
             return self.get_environment_specifier_by_name(source=source, name=name)
 
+    def get_environment_exporters(self) -> dict[str, CondaEnvironmentExporter]:
+        """
+        Returns a mapping from export format name to environment exporter.
+        """
+        return {
+            exporter.handler().format: exporter
+            for exporter in self.get_hook_results("environment_exporters")
+            if exporter.handler().format
+        }
```

Comment:
```suggestion
    def get_environment_exporters(self) -> Iterable[CondaEnvironmentExporter]:
        """
        Yields all detected environment exporters.
        """
        for exporter in self.get_hook_results("environment_exporters"):
            yield exporter
```

As per https://github.com/conda/conda/pull/14886/files/01d0592f055c85d7d8548e5b9065731434178183#r2171718782